deploy/helm/flightctl/templates/flightctl-cli-artifacts-deployment.yaml

```diff
@@ -0,0 +1,33 @@
+{{ if .Values.cliArtifacts.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    flightctl.service: flightctl-cli-artifacts
+  name: flightctl-cli-artifacts
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      flightctl.service: flightctl-cli-artifacts
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        flightctl.service: flightctl-cli-artifacts
+    spec:
+      containers:
+        - name: flightctl-cli-artifacts
+          image: {{ .Values.cliArtifacts.image.image }}:{{ default .Chart.AppVersion .Values.cliArtifacts.image.tag }}
+          imagePullPolicy: {{ .Values.cliArtifacts.image.pullPolicy }}
+          env:
+            - name: CLI_ARTIFACTS_BASE_URL
+              value: "https://www.example.com/"
+          ports:
+            - containerPort: 80
```

Comment:
would it make sense for the port to be variable?

Reply:
yep, done

Reply:
done